pkgs/tools/filesystems/juicefs/default.nix

```diff
@@ -1,10 +1,10 @@
 { lib
-, buildGoModule
+, buildGo121Module
```

Comment:
Please add a comment here to remind others to remove this when bumping version

Reply:
I added a note below the inputs, hope thats ok. if not lmk